src/source/Utils/ExponentialBackoffUtils.c

```diff
@@ -0,0 +1,221 @@
+#include "ExponentialBackoffUtils.h"
+#include "../Include_i.h"
+
+STATUS getDefaultExponentialBackOffConfig(PExponentialBackoffConfig* ppPExponentialBackoffConfig) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PExponentialBackoffConfig pExponentialBackoffConfig = NULL;
+
+    CHK(ppPExponentialBackoffConfig != NULL, STATUS_NULL_ARG);
+
+    pExponentialBackoffConfig = (PExponentialBackoffConfig) MEMALLOC(SIZEOF(ExponentialBackoffConfig));
```

Comment:
MEMCALLOC might be a better choice to zero out the struct.

Reply:
Sure. Changed at all places.